packages/react/src/avatar-group/AvatarGroup.tsx

```diff
@@ -29,16 +38,31 @@ export const AvatarGroup = forwardRef<HTMLDivElement, AvatarGroupProps>(
         {overflowChildren.length > 0 && (
           <HoverCard>
             <HoverCardTrigger asChild>
-              <Avatar colorScheme="gray" size={size}>
+              <Avatar colorScheme="neutral" size={size}>
                 +{overflowChildren.length}
               </Avatar>
             </HoverCardTrigger>
-            <HoverCardContent flexDirection="row">
-              {overflowChildren.map((child, index) => (
-                <Box asChild key={index}>
-                  {child}
-                </Box>
-              ))}
+            <HoverCardContent flexDirection="column" gap="xs">
+              {overflowChildren.map((child, index) =>
+                isValidElement<
+                  ComponentPropsWithRef<typeof Avatar> &
+                    ComponentPropsWithRef<typeof Tooltip>
+                >(child) ? (
+                  <Flex flexDirection="row" gap="4" key={index}>
+                    <AvatarGroupContext.Provider value={{ size }}>
```

Comment:
let's change the size to `md` for the popover
```suggestion
                    <AvatarGroupContext.Provider value="md">
```